lib/workers/repository/index.ts

```diff
@@ -30,7 +30,7 @@ export async function renovateRepository(
   logger.trace({ config });
   let repoResult: ProcessResult;
   try {
-    await fs.ensureDir(config.localDir);
+    await gitfs.ensureLocalDir(config.localDir);
```

Comment:
Wouldn't this join the localDir to itself? `const localDirName = join(localDir, dirName);`

Reply:
The global worker calls `setUtilConfig` which in turn passes `localDir` to `util/gitfs/fs`. So either we ensure the dir then and remove the above, or we change the above to be `gitFs.ensureLocalDir('.')`? The former seems easier.